CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
-# Release 1.41.0 09-10-2019
-
-- Replace oauth2client with google-auth (#706)
-- Update doc for secure image using custom nginx config (#703)
+# Release 1.42.0 23-10-2019
+
+- Update googleapis with latest CheckError codes (#722)
+- Fix e2e echo test failure (#721)
```

Comment:
remove these lines with tests. 4, 6, and 7

Reply:
Done